packages/vitest/src/reporters/console.ts

```diff
@@ -77,14 +77,26 @@ export class ConsoleReporter implements Reporter {
       }
     }
 
+    const errorMap = new Map<string | undefined, Test[]>()
+
     if (failedTests.length) {
       this.ctx.error(c.red(divider(c.bold(c.inverse(` Failed Tests ${failedTests.length} `)))))
       this.ctx.error()
       for (const test of failedTests) {
-        this.ctx.error(`${c.red(c.bold(c.inverse(' FAIL ')))} ${getFullName(test)}`)
-        await printError(test.result?.error, this.ctx)
+        const error = test.result?.error as Error
+        if (errorMap.get(error.stack))
+          errorMap.get(error.stack)!.push(test)
+        else
+          errorMap.set(error.stack, [test])
+      }
+      for (const [error, tests] of errorMap) {
```

Comment:
The error here becomes the stack string instead of the real error object 👀

Reply:
Oh, you're right! what's your suggestion here?

Reply:
oh i found an idea, let me push